ansible/roles/compute_init/files/compute-init.yml

```diff
@@ -109,17 +110,33 @@
         - meta: end_play
       when: not hostvars_stat.stat.exists
 
-    - name: Load hostvars from NFS
+    - name: Sync /mnt/cluster to /tmp
+      become_user: slurm
+      synchronize:
+        src: "/mnt/cluster/"
+        dest: "/tmp/cluster/"
```

Comment:
I don't really like using /tmp because I'm not sure files here are always going to be "temporary". I think it'd be better to use [/var](https://www.pathname.com/fhs/pub/fhs-2.3.html#THEVARHIERARCHY).